### PR TITLE
feat: Enable GCS FUSE driver addon

### PR DESCRIPTION
**Pull request by annirudh**

### CODE REVIEW

File: modules/app_gke/variables.tf [hunk 1]
```diff
@@ -65,6 +65,11 @@ variable "enable_private_gke_nodes" {
   description = "Enable private nodes for the GKE cluster."
 }
 
+variable "enable_gcs_fuse_csi_driver" {
```
Comment (danielpanzella) [inline modules/app_gke/variables.tf:68]: should we default this to false

Comment (annirudh) [reply modules/app_gke/variables.tf:68]: Omg definitely. Sorry, i meant to disable by default

Comment (annirudh) [reply modules/app_gke/variables.tf:68]: Fixed

**Review by vanpelt — APPROVED** [review]
🔥

**Comment by jsbroks** [comment]
This PR is included in version 6.6.0 :tada:

**Pull request merged**